bindings/nodejs/lib/types/models/api/block-response.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { BlockId, SlotCommitment, SlotIndex } from '../../block';
+
+/**
+ * Block id response.
+ */
+export interface IBlockIdResponse {
+    /**
+     * The block id.
+     */
+    blockId: BlockId;
+}
+
+/**
+ * Information that is used to attach a block in the network.
+ * Response of GET /api/core/v3/blocks/issuance
+ */
+export interface IssuanceBlockHeaderResponse {
```

Comment:
I know looks a bit weird, but for consistency this should also have an I prefix
```suggestion
export interface IIssuanceBlockHeaderResponse {
```

Reply:
There are more such instances, but didn't want to blow up the PR again since it's not really TIP-48 related but rather JS coding practices/style. Still want me do it (and also for the other interfaces)?

Reply:
Definitely not the other interfaces in this PR, this should be done separated

Reply:
Created an issue: #2044